configs/seqr-main.toml

```diff
@@ -0,0 +1,38 @@
+[workflow]
+access_level = 'full'
+datasets = [
```

Comment:
@illusional I wonder if we could also reflect these dependent datasets in metamist?